resources/example-configs/ktx/configs/usermodes/end2.cfg

```diff
@@ -0,0 +1,5 @@
+timelimit 3
+deathmatch 4
+set k_overtime 2
+set k_disallow_weapons 80		// disallow gl + lg
+set k_noitems 1				// disable items
```

Comment:
end2 lacks resources/example-configs/ktx/bots/maps/end2.bot ... ought to be a suboptimal experience?